vars/cloudFoundryCreateServiceKey.groovy

```diff
@@ -8,103 +9,39 @@ import groovy.transform.Field
 
 import static com.sap.piper.Prerequisites.checkScript
 
-@Field String STEP_NAME = 'cloudFoundryCreateServiceKey'
+@Field def STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/cloudFoundryCreateServiceKey.yaml'
```

Comment:
by now you could use the convenience step which came in with https://github.com/SAP/jenkins-library/pull/1254

Reply:
Latest commit already uses the convenience step if you relate to `@Field def STEP_NAME = getClass().getName()`